src/webview/html/messageAsHtml.js

```diff
@@ -54,7 +63,9 @@ const messageReactionListAsHtml = (
   const htmlList = aggregateReactions(reactions, ownUserId).map(r =>
     messageReactionAsHtml(r, allImageEmojiById),
   );
-  return template`<div class="reaction-list">$!${htmlList.join('')}</div>`;
+  return template`<div class="reaction-list">$!${htmlList.join(
+    '',
+  )} $!${addReactionButtonAsHtml()}  </div>`;
```

Comment:
nit: there's extra spaces here, as well as in the definition of `addReactionButtonAsHtml`. In general it's best to avoid having extra spaces or newlines in HTML text -- it's fine inside the tag syntax, after a `<` before the `>`, but avoid it in the literal text that comes after a `>` until the next tag's `<`. When it's in literal text it's part of what gets displayed, and can lead to hard-to-debug glitches in the layout.